### PR TITLE
fix test by changing the expect to be a single word to make the check more accurate

### PR DESCRIPTION
**Pull request by ShellyKa13**
Signed-off-by: Shelly Kagan <skagan@redhat.com>

**What this PR does / why we need it**:
This PR is a possible fix to the problematic test which is currently in quarantine

**Special notes for your reviewer**:
**_I remove the quarantine label to try and test and retest the fix several time in the PR, will return the quarantine label before the merge_**

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/storage/restore.go [hunk 1]
```diff
@@ -745,8 +745,8 @@ var _ = SIGDescribe("[Serial]VirtualMachineRestore Tests", func() {
 				}, 360, 10).Should(Succeed())
 
 				b := append([]expect.Batcher{
-					&expect.BSnd{S: "cat /var/run/resize.rootfs | grep resized\n"},
-					&expect.BExp{R: "resized successfully"},
+					&expect.BSnd{S: "cat /var/run/resize.rootfs | grep -q resized && echo 'pass'\n"},
+					&expect.BExp{R: console.RetValue("pass")},
```
Comment (rmohr) [inline tests/storage/restore.go:749]: Despite all the attempts to explain me the original  issue on the last PR where this go fixed, I still don't understand why only this test is affected. :D

 Does it have to do with a cirros start from a snapshot vs from a unmodified base image?

Comment (ShellyKa13) [reply tests/storage/restore.go:749]: The difference is that this test runs with a block storage vs filesystem, and with the block storage there is some resize happening in the bot of the vm(its before doing anything in the test before snapshot and the rest)

Comment (rmohr) [reply tests/storage/restore.go:749]: Does it happen with the alpine image too? Maybe that is an easy fix to just switch the image?

**Comment by ShellyKa13** [comment]
running the test again to make sure the fix is consistent
/test  pull-kubevirt-e2e-k8s-1.17-rook-ceph

**Pull request closed without merging**